.jenkins

```diff
@@ -414,11 +414,7 @@ pipeline {
                                 -DCMAKE_CXX_FLAGS=-Werror \
                                 -DCMAKE_CXX_STANDARD=17 \
                               .. && \
-                              make -j8 && ctest --verbose && \
-                              cd ../.. && \
-                              cmake -B build_cmake_installed_different_compiler/build -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_CXX_FLAGS=-Werror -DCMAKE_CXX_STANDARD=17 build_cmake_installed_different_compiler && \
-                              cmake --build build_cmake_installed_different_compiler/build --target all && \
-                              cmake --build build_cmake_installed_different_compiler/build --target test'''
+                              make -j8 && ctest --verbose'''
```

Comment:
Why did you change this?

Reply:
Using different compilers with `Kokkos` is problematic and doesn't work with `OpenMP` at all since we are using different flags.

Reply:
Please elaborate why it is ok to drop that test.

Reply:
It's just dysfunctional for `OpenMP` right now (which is the backend this CI configuration was set up to use in the first place) and it's not clear to me if this is something we want to address and if it's worth it to test a different configuration with a (significantly) different compiler than `Kokkos` was configured with.